core/domain/exp_domain.py

```diff
@@ -2190,6 +2211,10 @@ def _convert_states_v27_dict_to_v28_dict(cls, states_dict):
             }
         return states_dict
 
+    @classmethod
+    def _convert_states_v28_dict_to_v29_dict(cls, states_dict):
+        return states_dict
```

Comment:
This function should add the image_assets_mapping field to states right?

Reply:
Added full functionality.
Done PTAL!